CRM/Core/BAO/CustomGroup.php

```diff
@@ -937,6 +937,18 @@ public static function getTitle($id) {
     return self::getAll()[$id]['title'] ?? NULL;
   }
 
+  /**
+   * Get custom group details for a group. Legacy function for backwards compatibility.
+   * @deprecated Legacy function
+   *
+   * @see CRM_Core_BAO_CustomGroup::getAll()
+   * for a better alternative.
+   */
+  public static function &getGroupDetail($groupId = NULL, $searchable = FALSE, &$extends = NULL, $inSelector = NULL) {
+    CRM_Core_Error::deprecatedFunctionWarning('getCustomGroupDetail');
+    return self::getGroupDetail($groupId, $extends, $inSelector);
```

Comment:
@mlutfy isn't this should be  self::getCustomGroupDetail(...) instead of self::getGroupDetail(...) here ?

Reply:
@monishdeb yikes, nice catch, fixed - thanks!